### ISSUE TITLE
Make public/ folder configurable via env

### ISSUE THREAD

**Issue by razvanphp** [dunglas/frankenphp#765]
### Describe you feature request

We tried frankenphp official docker image with Yii2 framework, but our public folder is called `web` instead of `public`.

Can we make [this](https://github.com/dunglas/frankenphp/blob/main/caddy/frankenphp/Caddyfile#L31) configurable via some ENV variable in the official image?

I could probably override it in my own Dockerfile and Caddyfile, but it would be beneficial for other users too.

Thank you for this great piece of software!
R

**Comment by razvanphp** [comment]
https://github.com/dunglas/frankenphp/blob/12fb11eeaddb0ff63fc8e521c296659d54a83542/caddy/frankenphp/Caddyfile#L31

Maybe we could change it to `root * {$SERVER_ROOT:public/} `?

**Comment by dunglas** [comment]
Wouldn't it be better to override the Caddyfile in a custom stage?

**Comment by razvanphp** [comment]
Indeed, but it would be nice to have one-liner working with the official docker image for local development.

Should be backwards compatible as the default will still be `public/`

**Comment by dunglas** [comment]
Ok, why not!

Do you want to open a PR?
